front/app/components/EventCard/EventInformation/index.tsx

```diff
@@ -113,6 +114,31 @@ const EventInformation = ({
               </Text>
             </Box>
           )}
+          {onlineLink && (
+            <Box display="flex" mb="12px">
+              <Icon
+                my="auto"
+                fill={colors.coolGrey300}
+                name="link"
+                ariaHidden
+                mr="8px"
+              />
+              <Text
+                m="0px"
+                color={'coolGrey700'}
```

Comment:
Just a guess :)
```suggestion
                color={colors.coolGrey700}
```
or
```suggestion
                color="coolGrey700"
```

Reply:
Fixed! Thanks :)